<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup - Pilocik</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
	background-image: url(':/images/background.png');
	background-repeat: no-repeat;
} 

#tab1,#tab2,#tab3,#tab4,#tab5,#tab6{
	width:173px;
	height:65px;
	padding: 0;
	padding-left: 15px;
	font-size: 16px;
	font-family: 'Trebuchet MS', Tahoma, Verdana;
	font-weight: bold;
	color: rgb(255, 255, 255);
	text-align: left;
	background-image: url(':/images/inactiveTab.png');
	background-repeat: no-repeat;
	background-color: transparent;
	background-position: right center;
}

#exitBtn{
	width: 63px;
	height: 26px;
	background-image: url(':/images/exit.png');
	background-repeat: no-repeat;
	background-color: transparent;
}

#exitBtn:hover{
	background-image: url(':/images/exit_hover.png');
}

#minimalizeBtn{
	width: 33px;
	height: 26px;
	background-image: url(':/images/minimalize.png');
	background-repeat: no-repeat;
	background-color: transparent;
}

#minimalizeBtn:hover{
	background-image: url(':/images/minimalize_hover.png');
}

#header{
	font-weight: bold; 
	font-size: 16px; 
	font-family:'Trebuchet MS', Tahoma, Verdana;
	color: #fe7e00;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headerFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>103</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>103</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="exitBtn">
       <property name="geometry">
        <rect>
         <x>731</x>
         <y>3</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="minimalizeBtn">
       <property name="geometry">
        <rect>
         <x>695</x>
         <y>3</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="menuFrame">
         <property name="maximumSize">
          <size>
           <width>194</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>23</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="tab1">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>News</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/News.png</normaloff>:/images/News.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tab2">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Map
download</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/Map.png</normaloff>:/images/Map.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tab3">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Install</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/Hard drive.png</normaloff>:/images/Hard drive.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tab4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/System.png</normaloff>:/images/System.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tab5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Styles</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/paint tools.png</normaloff>:/images/paint tools.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tab6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Plugins</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/Download crate.png</normaloff>:/images/Download crate.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="contentFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>22</number>
          </property>
          <property name="bottomMargin">
           <number>23</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
